### PR TITLE
Add missing tests for codeGen

### PR DESCRIPTION
**Pull request by ushirask**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: test/ll/Vwhile05.ll [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  store i1 0, i1* %_0, align 8
+  ret void
+}
+define internal void @_B_printWhile () {
+  %_0 = alloca i1, align 8
+  br label %L1
+  call void @_Bio__println (i64 0)
+  store i1 0, i1* %_0, align 8
+  ret void
+L1:
```
Comment (ushirask) [inline test/ll/Vwhile05.ll]: @jclark this is the ll for while false statement. Is this intended as while false is redundant?

File: compiler/testSuite/Ecall05.bal [hunk 2]
```diff
@@ -0,0 +1,8 @@
+public function main() {
+    int x = 1;
+    foo(x); // @error
+}
+
+type foo(
```
Comment (jclark) [inline compiler/testSuite/Ecall05.bal]: That's not correct `type` definition syntax

File: compiler/testSuite/Vbooleanorder01.bal [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import ballerina/io;
+
+public function main() {
+    printComp(); //@output 1
+}
+
+function printComp() {
+    if (true > false) {
```
Comment (jclark) [inline compiler/testSuite/Vbooleanorder01.bal]: No parentheses here please

File: compiler/testSuite/Vifelse07.bal [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import ballerina/io;
+
```
Comment (jclark) [inline compiler/testSuite/Vifelse07.bal:2]: Isn't this already handled by Vifelse06?

File: compiler/testSuite/Vbooleanorder01.bal [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import ballerina/io;
```
Comment (jclark) [inline compiler/testSuite/Vbooleanorder01.bal:1]: What does this add to Vbooleanorder00?

File: compiler/testSuite/Ewhile01.bal [hunk 6]
```diff
@@ -0,0 +1,5 @@
+public function main() {
```
Comment (jclark) [inline compiler/testSuite/Ewhile01.bal]: This should be a U case

File: test/ll/Vwhile05.ll (hunk 1 above)
Comment (jclark) [reply test/ll/Vwhile05.ll]: Have you tested the output? I don't think that's valid `ll` syntax.

If you find bugs in the code generation, please create a separate issue.

Edited: I just checked: that's no valid `ll` syntax, so it's a bug. No need to report this one.

**Review by jclark — CHANGES_REQUESTED** [review]

File: compiler/testSuite/Vifelse07.bal (hunk 4 above)
Comment (ushirask) [reply compiler/testSuite/Vifelse07.bal:2]: directly using the boolean value as if condition than using a stored value takes a different code path so this was added for code coverage

File: compiler/testSuite/Vbooleanorder01.bal (hunk 5 above)
Comment (ushirask) [reply compiler/testSuite/Vbooleanorder01.bal:1]: directly comparing the boolean values without assigning is tested here for code coverage

File: test/ll/Vwhile05.ll (hunk 1 above)
Comment (jclark) [reply test/ll/Vwhile05.ll]: I fixed the bug for `while false`. You can remove this one.

File: compiler/testSuite/Uwhile00.bal [hunk 7]
```diff
@@ -0,0 +1,5 @@
+public function main() {
```
Comment (jclark) [inline compiler/testSuite/Uwhile00.bal:3]: A U case like a V case. With a println @output etc.  When I implement it, I should be able to rename U to V and it should work.

**Review by jclark — CHANGES_REQUESTED** [review]

**Pull request merged**